sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -62,13 +64,15 @@
 from apache_beam.runners.portability.fn_api_runner.translations import only_element
 from apache_beam.runners.portability.fn_api_runner.worker_handlers import WorkerHandlerManager
 from apache_beam.transforms import environments
-from apache_beam.utils import profiler
 from apache_beam.utils import proto_utils
 from apache_beam.utils import thread_pool_executor
+from apache_beam.utils.profiler import Profile
```

Comment:
I changed this import because `profiler` was conflicting with some local variable names, and this was the simplest resolution.